### PR TITLE
Update action documentation to include new input data

### PR DESCRIPTION
**Pull request by kmh287**
Updating documentation based on the changes in #8569 

Documentation for features requested in #7765 

/to @aghassemi @bpaduch

### CODE REVIEW

File: spec/amp-actions-and-events.md [hunk 1]
```diff
@@ -102,10 +102,36 @@ Including: `input[type=radio]`, `input[type=checkbox]` and `select`.
   <tr>
     <th>Event</th>
     <th>Description</th>
+    <th>Data</th>
   </tr>
   <tr>
     <td>change</td>
     <td>Fired when the value of the element is changed.</td>
+    <td> Various, see below.</td>
```
Comment (aghassemi) [inline spec/amp-actions-and-events.md]: nit: remove extra space

File: spec/amp-actions-and-events.md [hunk 2]
```diff
@@ -102,10 +102,36 @@ Including: `input[type=radio]`, `input[type=checkbox]` and `select`.
   <tr>
```
Comment (aghassemi) [inline spec/amp-actions-and-events.md]: There is a line above that says: "Including: `input[type=radio]`, `input[type=checkbox]` and `select`." Let's add `range` to it explicitly as well.

Comment (kmh287) [inline spec/amp-actions-and-events.md]: ok

File: spec/amp-actions-and-events.md (hunk 1 above)
Comment (kmh287) [inline spec/amp-actions-and-events.md]: done

**Pull request merged**